quipucords/quipucords/settings.py

```diff
@@ -283,6 +283,10 @@
     },
 }
 
+SCAN_DATA_LOG_BASENAME = os.path.join(BASE_DIR, 'scan_data_log')
```

Comment:
This is ok as a default, but it should be configurable like the other logs so it can end up in the same directory as the logs in the container setup. See line 205.

Reply:
Done.

Reply:
Right now it's a separate environment variable, to match the other log. What do you think of the idea of either having a LOG_DIRECTORY variable where all the logs go (and still allowing overrides for each type of log) or making this default to the same directory as DJANGO_LOG_FILE but with a different name?

Reply:
I think its fine to do it with the LOG_DIRECTORY env. I didn't see a separate env var set for it.
Is there a variable to disable this function?

Reply:
I didn't upload my latest changes, sorry. I'll push again in a moment with a LOG_DIRECTORY, which can be overridden for the individual log files.

For disabling, how about setting the scan data log file path to "" disables it?

Reply:
I'd prefer an explicit DISABLE env ans this will be more intuitive for a user than setting an empty string. If they for some reason want to disable this log capture.